py4DSTEM/io/read.py

```diff
@@ -6,19 +6,15 @@
 import h5py
 
```

Comment:
This looks correct and doesn't raise any import errors but I haven't checked the against un/accepted file types